tests/test_google_analytics_start_date.py

```diff
@@ -81,6 +81,7 @@ def test_run(self):
                 expected_start_date_2 = self.start_date_2
 
                 # collect information for assertions from syncs 1 & 2 base on expected values
+                # Exclude the last 2 days to avaoid instability in the tests- TDL-22494
```

Comment:
This PR is just a one line comment update.  Seems like the previous commit to master wasn't reverted.  Not sure if it is worth reverting it now or just merging this new PR on top since it seems to pass as is.